### PR TITLE
prepare release 3.0.3

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

**Comment by chaudum** [comment]
retest this please

File: blackbox/docs/appendices/release-notes/3.0.3.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  for s3 repositories, if they are not passed as parameters to the
+  ``CREATE REPOSITORY`` SQL statement.
+
+- Implemented a thread-utilization down-scaling logic which dynamically adapts
+  the number of threads used for ``SELECT`` queries to avoid running into
+  ``RejectedExcecution`` errors if there are many shards per node involved in
+  the queries.
+
+- Changed internals for DELETE and UPDATE queries which should generally result
+  in a performance increase for queries which only match a subset of the rows
```
Comment (autophagy) [inline blackbox/docs/appendices/release-notes/3.0.3.rst]: just a personal preference, but I'd reword this to remove "internals". Just saying we improved performance in these cases would be fine & reads better

File: blackbox/docs/appendices/release-notes/3.0.3.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+- Improved performance for expressions involving literal type conversions,
+  e.g. ``select count(*) from users group by name having max(bytes) = 4``.
+
+- Fixed an issue which could result in lost entries at the ``sys.jobs_log`` and
+  ``sys.operations_log`` tables when the related settings are changed while
+  entries are written.
+
+- Fixed a dependecy issue with the bundled ``crash`` that caused the app not
+  being able to connect to the server.
```
Comment (autophagy) [inline blackbox/docs/appendices/release-notes/3.0.3.rst]: "not being able" -> "to be unable"

**Comment by chaudum** [comment]
Thanks for the comments @autophagy I amended the changes.
